### PR TITLE
sokgazetesi.com.tr

### PR DESCRIPTION
**Pull request by gork7777**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

https://github.com/AdguardTeam/AdguardFilters/issues/139304

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -422,7 +423,7 @@ vidmoxy.com,rapidvid.net##.rkbg
 ||yenimarmaragazetesi.com/dimg/banner/
 ||filmpaf.com/1/1.gif
 ||diziyoucdn.xyz/*.mp4
-/get-ad|$domain=bursahayat.com.tr|bursainternetgazetesi.com|bursasporum.com|bursaport.com|igfhaber.com|lifebursa.com|lodoshaber.com|sozbursa.com|yenidonem.com.tr|webursa.com|bandirmayasam.com.tr|bbursa.com|bursa5n1k.com|bursasehrengiz.com|dijitalhaber.com.tr|fotospor.com|gencfm.com.tr|haberindogrusu.com|kadinveekonomi.com|kaleninsesi.com|korkutelimanset.com|marmaratv.com.tr|otoritehaber.com|sokgazetesi.com.tr|bizimgazete.com.tr|bursaekspress.com|bilginhaberci.com
+/get-ad|$domain=sokgazetesi.com.tr|bursahakimiyet.com.tr|bursahayat.com.tr|bursainternetgazetesi.com|bursasporum.com|bursaport.com|igfhaber.com|lifebursa.com|lodoshaber.com|sozbursa.com|yenidonem.com.tr|webursa.com|bandirmayasam.com.tr|bbursa.com|bursa5n1k.com|bursasehrengiz.com|dijitalhaber.com.tr|fotospor.com|gencfm.com.tr|haberindogrusu.com|kadinveekonomi.com|kaleninsesi.com|korkutelimanset.com|marmaratv.com.tr|otoritehaber.com|sokgazetesi.com.tr|bizimgazete.com.tr|bursaekspress.com|bilginhaberci.com
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt:426]: duplicate of `sokgazetesi.com.tr`

**Comment by Alex-302** [comment]
Also auto-refresh each 60 seconds can be disabled
`sokgazetesi.com.tr#%#AG_onLoad(function(){document.querySelector('meta[http-equiv="refresh"]')&&setTimeout(function(){window.stop()},6E3)});`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (gork7777) [reply TurkishFilter/sections/specific.txt:426]: @Alex-302 Which one is duplicate?

**Comment by gork7777** [comment]
> Also auto-refresh each 60 seconds can be disabled `sokgazetesi.com.tr#%#AG_onLoad(function(){document.querySelector('meta[http-equiv="refresh"]')&&setTimeout(function(){window.stop()},6E3)});`

Should be on annoyances or tweaks?

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply TurkishFilter/sections/specific.txt:426]: `sokgazetesi.com.tr`
image

**Comment by Alex-302** [comment]
> Should be on annoyances or tweaks?

general_extensions.txt

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**